### DISCUSSION TITLE
It would be so cool to be able to make Copilot the co-authors of commits

### DISCUSSION THREAD

**Discussion by p-cauty** [community/community#7222] [Copilot Conversations]
Hello,

I don't know is this topic have already been discussed (I looked for it but I didn't find anything, maybe I used wrong keywords), but it would be so great if the Copilot plugin could annotate lines written by Copilot with its name, and to add it as the co-author of the commits where it contributed.

That way, when git blaming code months later, instead of asking ourselves "where the hell did that code come from!?" we can easily see that it has been wrote by Copilot (plus the cool side of having it as the official pair programmer in commits list).

I am sorry if my English is not perfect and I hope I'm not the only one wanting this 😁👍🏼

Anyway, Copilot is awesome! Bye 👋🏼

**Comment by dlucian** [comment]
I think it's a *co*-pilot for a reason. The ultimate responsibility is the pilot's ;-)
